Source/WebCore/style/RuleSet.h

```diff
@@ -115,6 +115,7 @@ class RuleSet : public RefCounted<RuleSet> {
     bool hasAttributeRules() const { return !m_attributeLocalNameRules.isEmpty(); }
     bool hasUserAgentPartRules() const { return !m_userAgentPartRules.isEmpty(); }
     bool hasHostPseudoClassRulesMatchingInShadowTree() const { return m_hasHostPseudoClassRulesMatchingInShadowTree; }
+    bool mustCheckShadowHostUniversalBucket() const { return m_mustCheckShadowHostUniversalBucket; }
```

Comment:
maybe name this the same style as a the rest (hasSomething) instead of what must be done.